scene/2d/tile_map.cpp

```diff
@@ -1190,6 +1191,15 @@ void TileMap::set_light_mask(int p_light_mask) {
 	}
 }
 
+void TileMap::set_blend_mode(BlendMode p_blend_mode) {
+
+	CanvasItem::set_blend_mode(p_blend_mode);
+	_recreate_quadrants();
+	_update_dirty_quadrants();
```

Comment:
Hm, I'm not sure you need to call `_update_dirty_quadrants` here at all -- also, I think you should not call the same frame, but better defer it. E.g. Imagine someone does the following:
```gdscript
func update_effects():
  effect_tilemap.set_blend_mode(BLEND_MODE_ADD)
  if player.is_dead():
    effect_tilemap.set_blend_mode(BLEND_MODE_MULTIPLY)
```
Then, you would update quadrants twice, rather than just once.